Pyx.Uwp.App/MainPage.xaml.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Pyx.Client;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.System.Threading;
+using Windows.UI.Core;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+
+// The Blank Page item template is documented at https://go.microsoft.com/fwlink/?LinkId=402352&clcid=0x409
+
+namespace Pyx.Uwp.App
+{
+    /// <summary>
+    /// An empty page that can be used on its own or navigated to within a Frame.
+    /// </summary>
+    public sealed partial class MainPage : Page
+    {
+        private readonly ApiHelper _apiHelper = new ApiHelper("http://{LocalIP}:5002");
+        private ThreadPoolTimer _timer;
+        private int _currentId = 1;
+
+        public MainPage()
+        {
+            this.InitializeComponent();
+
+            PollForNextId();
+        }
+
+        private void PollForNextId()
```

Comment:
Instead of using a timer, this method could be made async. The Next_Click method could be made async so that it could await the call. MainPage could call this method without await or there is are "page loaded" type events which could be used IIRC.